application/services/Evaluation.php

```diff
@@ -950,17 +949,18 @@ public function getEvaluateReportsInPdf ($shipmentId, $sLimit, $sOffset) {
             ->order("sorting_unique_identifier");
         if (isset($sLimit) && isset($sOffset)) {
             $sql = $sql->limit($sLimit, $sOffset);
-		}
+        }
         $shipmentResult = $db->fetchAll($sql);
         $previousSixShipmentsSql = $sql = $db->select()
             ->from(array('s' => 'shipment'), array(
                 's.shipment_id',
                 's.shipment_code',
                 's.shipment_date'))
-            ->join(array('spm' => 'shipment_participant_map'), 's.shipment_id=spm.shipment_id', array('mean_shipment_score' => new Zend_Db_Expr("AVG(IFNULL(spm.shipment_score, 0) + IFNULL(spm.documentation_score, 0))")))
+            ->join(array('spm' => 'shipment_participant_map'), 's.shipment_id=spm.shipment_id', array('mean_shipment_score' => new Zend_Db_Expr("CASE WHEN spm.is_pt_test_not_performed=\"yes\" THEN AVG(IFNULL(spm.shipment_score, 0) + IFNULL(spm.documentation_score, 0)) ELSE 0 END")))
             ->where("s.is_official = 1")
             ->where("s.shipment_id <= ".$shipmentId)
             ->group('s.shipment_id')
+            ->order("RIGHT(shipment_code,5) DESC")

```

Comment:
It turns out that the reason these were badly ordered was because the 2017-B panel had a date_shipped date in 2016. I've corrected the data and this is now resolved. You can revert this change.